cli/src/commands/config.rs

```diff
@@ -174,6 +173,113 @@ fn toml_escape_key(key: String) -> String {
     toml_edit::Key::from(key).to_string()
 }
 
+fn to_toml_value(value: &config::Value) -> Result<toml_edit::Value, config::ConfigError> {
```

Comment:
(No action needed) I'm not sure why `config::Value` does not implement `Serialize`. I thought for a second of contributing it, but it seems the maintainer of `config` is already overwhelmed.